orttraining/orttraining/test/training_ops/cuda/softmax_test.cc

```diff
@@ -44,8 +44,9 @@ static void TestSoftmaxGrad(const std::vector<int64_t>& dY_dims,
 
   // create rand inputs
   RandomValueGenerator random{};
-  std::vector<float> dY_data = random.Uniform<float>(dY_dims, -10.0f, 10.0f);
-  std::vector<float> Y_data = random.Uniform<float>(Y_dims, -10.0f, 10.0f);
+  std::vector<float> dY_data = random.Uniform<float>(dY_dims, 0.02f, 1.02f);
```

Comment:
nit. the offset of 0.02 is only need for Y_data. 
Please also keep a comment explaining why we need this

Reply:
sorry, that was on accident. thanks.